src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -35,17 +37,46 @@ object Lorre extends App with LazyLogging {
       sys.exit(1)
     }
 
-  private val conf = ConfigFactory.load
-  private val sleepIntervalInSeconds = conf.getInt("lorre.sleepIntervalInSeconds")
-  private val feeUpdateInterval = conf.getInt("lorre.feeUpdateInterval")
-  private val purgeAccountsInterval = conf.getInt("lorre.purgeAccountsInterval")
+  //reads all configuration upstart, will only complete if all values are found
+  val CombinedConfiguration(lorreConf, tezosConf, sodiumConf, batchingConf) = {
```

Comment:
Maybe this can be moved to a ConfigUtil file? We want Lorre.scala to be minimal and do exactly what it needs to be run and be self explanatory with it's naming.